serverless.yml

```diff
@@ -236,3 +233,31 @@ functions:
     package:
       include:
         - functions/AutoRemediateEC-005.js
+    role: revokeSecurityGroupIngressRole
+
+resources:
+
+  Resources:
+
+    revokeSecurityGroupIngressRole:
```

Comment:
To keep it consistent, can you please change revokeSecurityGroupIngressRole to AutoRemediateEC-002Role

Reply:
I can but for this specific role, it belongs to a bunch of EC2 functions ( all unrestricted ones), so calling it EC-002Role is not completely right

Reply:
Well, what if a function needs to 1) DescribeResource1 2) UpdateResource1 3) UpdateResource2 4) etc.  In that case, it might become confusing to call the IAM Role (or the IAM Policy) DescribeResource1AndUpdateResource1AndUpdateResource2
Also, when you look at the Role on the IAM Management Console, the name would be explanatory if called AutoRemediateEC-002Role as you know it's related to Auto-Remediation and you also can easily figure out it's for EC2-002. Thoughts ?

Reply:
The idea is to implement the Principle Of Least Privilege (PoLP) and the permission associated to a lambda function should be regularly reviewed; And not necessarily by the person who's implementing the function in question ....

Reply:
that makes sense. but in this case, do you mean we need to have a seperate role and policy for each of the unrestricted access rules? they are practically the same thing though